examples/browser/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "browser",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@blaze-cardano/core": "^0.4.4",
+    "@blaze-cardano/query": "^0.2.15",
+    "@blaze-cardano/sdk": "^0.1.30",
+    "@blaze-cardano/tsconfig": "^0.0.2",
+    "@saibdev/blaze-provider": "0.1.6",
```

Comment:
I think this is from testing version, right?
should we point by path to the lib in the parent dir?